yml/apdb.yaml

```diff
@@ -2077,10 +2084,11 @@ tables:
     datatype: long
   primaryKey: "#DiaSource.diaSourceId"
   indexes:
-  - name: IDX_DiaSource_ccdVisitId
-    "@id": "#IDX_DiaSource_ccdVisitId"
+  - name: IDX_DiaSource_detectorVisit
+    "@id": "#IDX_DiaSource_detectorVisit"
```

Comment:
Maybe change the name to `IDX_DiaSource_visitDetector` to reflect the order of the columns in the index.

Reply:
Good catch, thanks! I've tried to be consistent about visit,detector wherever I can (e.g. in function signatures), but missed this one.